packages/scandipwa/src/route/CartPage/CartPage.style.scss

```diff
@@ -104,13 +104,12 @@
             display: flex;
             align-items: center;
             justify-content: center;
-            border: 1px solid var(--cart-page-divider-background);
+            border: 1px solid var(--cart-page-table-head-background);
```

Comment:
Since ``` --cart-page-divider-background ``` is used only here, better to set correct value for variable and not replace it.